hack/create_bmcs.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# This script reads BMC information in a config file and prepare VMs 
+# whose info match those config
+#
+set -eux
+
+REPO_ROOT=$(realpath "$(dirname "${BASH_SOURCE[0]}")/..")
+cd "${REPO_ROOT}" || exit 1
+
+CONFIG_FILE=$1
+NETWORK=${2:-"baremetal-e2e"}
+
+readarray -t BMCs < <(yq e -o=j -I=0 '.[]' "${CONFIG_FILE}")
```

Comment:
```suggestion
readarray -t BMCS < <(yq e -o=j -I=0 '.[]' "${CONFIG_FILE}")
```
Not a fan of mixed-case variables